pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -156,6 +166,14 @@ func (a *AgentClusterInstall) Load(f asset.FileFetcher) (bool, error) {
 		err = errors.Wrapf(err, "failed to unmarshal %s", agentClusterInstallFilename)
 		return false, err
 	}
+
+	if agentClusterInstall.Spec.Networking.NetworkType == "" {
+		// default NetworkType to OpenShiftSDN if unspecified
+		// https://docs.openshift.com/container-platform/4.11/installing/installing_aws/installing-aws-network-customizations.html#installation-configuration-parameters-network_installing-aws-network-customizations
+		logrus.Info("NetworkType is not specified in AgentClusterInstall. Defaulting to OpenShiftSDN.")
+		agentClusterInstall.Spec.Networking.NetworkType = "OpenShiftSDN"
```

Comment:
Is there some way to retrieve the default from elsewhere in the code so that if it changes we don't get this left behind?

Reply:
Found it! It's in pkg/types/defaults/installconfig.go
```
defaultNetworkTYpe = string(operv1.NetworkTypeOpenShiftSDN)).
```
I believe if unset we should just retrieve it from installconfig defaults.

Reply:
Now that I look closer, in the same go file, if we look at SetInstallConfigDefaults, there's logic that changes the SDN depending on topology and upstrea/downstream. Being able to consume that logic would be useful.

Reply:
Now that I look closer, in the same go file, if we look at SetInstallConfigDefaults, there's logic that changes the SDN depending on topology and upstrea/downstream. Being able to consume that logic would be useful.

Reply:
Good find. I will change it to use SetInstallConfigDefaults.